### PR TITLE
added locale api

### PR DESCRIPTION
**Pull request by acaldas**
https://github.com/ripe-tech/ripe-core/issues/3702

### CODE REVIEW

File: src/js/api/locale.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+ripe.Ripe.prototype.getLocale = function(values, locale, options, callback) {
+    values = typeof options === "string" ? [values] : values;
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" ? {} : options;
+    var url = this.url + "locale";
+    options = Object.assign(options, {
+        url: url,
+        method: "POST",
```
Comment (hugo-gomes) [inline src/js/api/locale.js]: this is not `GET` api?

File: src/python/ripe_demo/static/js/ripe.js [hunk 2]
```diff
@@ -1397,6 +1397,30 @@ if (typeof require !== "undefined") {
     var ripe = base.ripe;
 }
 
+ripe.Ripe.prototype.getLocale = function(values, locale, options, callback) {
```
Comment (joamag) [inline src/python/ripe_demo/static/js/ripe.js]: the name of this method should be `localeMultiple` to be in line with the name locale_m on the server side

File: src/js/api/locale.js [hunk 3]
```diff
@@ -5,8 +5,12 @@ if (typeof require !== "undefined") {
     var ripe = base.ripe;
 }
 
-ripe.Ripe.prototype.getLocale = function(values, locale, options, callback) {
-    values = typeof options === "string" ? [values] : values;
+ripe.Ripe.prototype.getLocale = function(value, locale, options, callback) {
+    return this.getLocaleMultiple(value, locale, options, callback);
+};
+
+ripe.Ripe.prototype.getLocaleMultiple = function(values, locale, options, callback) {
```
Comment (joamag) [inline src/js/api/locale.js]: just localeMultiple

File: src/js/api/locale.js [hunk 4]
```diff
@@ -5,8 +5,12 @@ if (typeof require !== "undefined") {
     var ripe = base.ripe;
 }
 
-ripe.Ripe.prototype.getLocale = function(values, locale, options, callback) {
-    values = typeof options === "string" ? [values] : values;
+ripe.Ripe.prototype.getLocale = function(value, locale, options, callback) {
```
Comment (joamag) [inline src/js/api/locale.js]: just locale

**Pull request merged**